packages/core/src/prompts/select-key.ts

```diff
@@ -0,0 +1,26 @@
+import Prompt, { PromptOptions } from './prompt';
+
+interface SelectKeyOptions<T extends { value: any }> extends PromptOptions<SelectKeyPrompt<T>> {
+	options: T[];
+}
+export default class SelectKeyPrompt<T extends { value: any }> extends Prompt {
+	options: T[];
+	cursor: number = 0;
+
+	constructor(opts: SelectKeyOptions<T>) {
+		super(opts, false);
+
+		this.options = opts.options;
+		const keys = opts.options.map(v => v.value[0]);
```

Comment:
Since you're lowercasing the char in the event payload, shouldn't these be lowercased as well? Or use a case-insensitive check like `.test(/.../i)`.

Reply:
Went ahead and lowercased these + lowercased in comparison.